test/test_worker.rb

```diff
@@ -0,0 +1,20 @@
+class TestWorker
```

Comment:
Could we put this in a `test/support` path to be clear that it's not a test? Confused me at first glance.